File: Week 1/Python/prog6.py
arr = [1, 2, 3, 4, 5]

is_sorted = all(arr[i] <= arr[i+1] for i in range(len(arr)-1))

if is_sorted:
    print("Array is sorted in ascending order.")
else:
    print("Array is NOT sorted in ascending order.")

File: Week 1/C programming/prog6.c
#include <stdio.h>

int main() {
    int arr[100], n, sorted = 1;

    printf("Enter number of elements: ");
    scanf("%d", &n);

    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    for (int i = 0; i < n - 1; i++) {
        if (arr[i] > arr[i + 1]) {
            sorted = 0;
            break;
        }
    }

    if (sorted)
        printf("Array is sorted in ascending order.\n");
    else
        printf("Array is NOT sorted in ascending order.\n");

    return 0;
}
